documentation/justfile

```diff
@@ -18,6 +19,48 @@ install:
 # Sphinx #
 ##########
 
+# Create a new project with proposal and implementation plans
+init-project name *ips:
+    #!/usr/bin/env bash
+    set -eo pipefail
+    TIMESTAMP=$(date '+%Y%m%d')
+    DATE=$(date '+%Y-%m-%d')
+    PROJECTFS="{{ replace(lowercase(name), ' ', '_') }}"
+    # Create project directory
+    mkdir "projects/proposals/$PROJECTFS"
+    # Create project directory index
+    sed \
+      -e '1,3d;$d' \
+      -e 's/{Project title}/{{ name }}/g' \
+      projects/templates/index.md > "projects/proposals/$PROJECTFS/index.md"
```

Comment:
Optionally, it would be nice if these commands logged the paths of the files they're creating.

Reply:
Great idea! Added in 14f0d3d.